src/ir/ty.rs

```diff
@@ -869,6 +874,60 @@ impl Type {
                                            parent_id,
                                            ctx);
             }
+            CXType_ObjCInterface => {
```

Comment:
Yeah, probably you don't want a new `itemid` for each Interface you parse, and I don't see any deduplication going on, so probably you should make this the actual container of your data (and use `cursor.definition()` to properly go to the actual definition, just in case you find a forward declaration before). You probably need to do roughly the same thing we do for `CXType_Record` (which is a plain struct), just storing most of the specific interface-related data you want.